npm install failed against angular version 2.0.0-beta.15

npm install failed again angular version 2.0.0-beta.15

```
npm WARN cannot run in wd portal-consumer@0.0.1 npm run typings && npm run build (wd=/opt/atlassian/bitbucketci/agent/build)
npm ERR! Linux 3.13.0-85-generic
npm ERR! argv "/root/.nvm/versions/node/v4.4.3/bin/node" "/root/.nvm/versions/node/v4.4.3/bin/npm" "install"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package angular2@2.0.0-beta.15 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer angular2-modal@0.1.3 wants angular2@2.0.0-beta.14
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "prepublish": "tsc --project publish && node make.js"
   },
   "peerDependencies": {
-    "angular2": "2.0.0-beta.14"
+    "angular2": "2.0.0-beta.15"
   },
   "devDependencies": {
     "async": "^1.5.2",
@@ -51,6 +51,6 @@
     "url-loader": "^0.5.6",
     "webpack": "^1.12.9",
     "webpack-dev-server": "^1.12.1",
-    "zone.js": "^0.6.6"
+    "zone.js": "^0.6.10"
   }
 }

